<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditDialog</class>
 <widget class="QDialog" name="EditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>418</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Calibri Light</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="15" column="1">
      <widget class="QLineEdit" name="lineEdit_Global"/>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="labelOther">
       <property name="text">
        <string>Other sales</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QLineEdit" name="lineEdit_Other"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="labelNA">
       <property name="text">
        <string>NA sales</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_Year"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelGenre">
       <property name="text">
        <string>Genre</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_Platform"/>
     </item>
     <item row="11" column="1">
      <widget class="QLineEdit" name="lineEdit_JP"/>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="lineEdit_EU"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_Publisher"/>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="labelGlobal">
       <property name="text">
        <string>Global sales</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelPlatform">
       <property name="text">
        <string>Platform</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_Genre"/>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="labelJP">
       <property name="text">
        <string>JP sales</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelPublisher">
       <property name="text">
        <string>Publisher</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="lineEdit_NA"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelName">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="labelEU">
       <property name="text">
        <string>EU sales</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelYear">
       <property name="text">
        <string>Year</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_Name"/>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Finish Edit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>released()</signal>
   <receiver>EditDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
